templates/systemd-networkd-vxlan.netdev.j2

```diff
@@ -0,0 +1,10 @@
+[NetDev]
+Name={{ item.device }}
+Kind=vxlan
+
+[VXLAN]
+Id={{ item.vni | default(vxlan_vni) }}
```

Comment:
docs suggest this should be `VNI`? https://www.freedesktop.org/software/systemd/man/systemd.netdev.html

Reply:
Yes I set this previously but I must have forgot to commit it. Explains the issues i was having with creating a new multinode as both were using the same vni. Will change. Thanks Mark!